### PR TITLE
feat(collapsible): remove deprecated styles from stylesheets

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
## Why

Remove deprecated styles $ca and $kz from stylesheets and replace with tokens.

## What

- Replace styles with tokens.

### CODE REVIEW

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss [hunk 1]
```diff
@@ -1,9 +1,8 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/shadow";
 @import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
```
Comment (mcwinter07) [inline draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:4]: Still one of my favorite things to see removed

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
-@import "~@kaizen/component-library/styles/border";
-@import "~@kaizen/component-library/styles/animation";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/animation";
 
 $heading-active-color: #f2f2f3;
```
Comment (mcwinter07) [inline draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:7]: Don't know if we want to do this in this PR but wonder if we should get Jason to take a look to see if there's a comparable color-gray token?

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss [hunk 3]
```diff
@@ -12,7 +11,7 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
 // We need a full border radius on this container element, then have classes
 // beneath to toggle nested borders off and on for different use cases.
 .container {
-  background-color: white;
+  background-color: $color-white;
```
Comment (mcwinter07) [inline draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:14]: Nice catch

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss [hunk 4]
```diff
@@ -64,16 +71,20 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
...
-  @include ca-padding(
-    $top: $ca-grid,
-    $end: $ca-grid,
-    $bottom: $ca-grid,
-    $start: $ca-grid
-  );
+  padding-top: $spacing-md;
+  padding-bottom: $spacing-md;
+  padding-right: $spacing-md;
+  padding-left: $spacing-md;
```
Comment (mcwinter07) [inline draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:81]: If this is all the same padding could we just use `padding: $spacing-md`? Possibly the same thing on ln 113

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss [hunk 5]
```diff
@@ -64,16 +71,20 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
...
+
+  &[dir="rtl"],
+  [dir="rtl"] & {
+    padding-right: $spacing-md;
+    padding-left: $spacing-md;
+  }
```
Comment (mcwinter07) [inline draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:87]: Possibly don't need this if the padding isn't changing in RTL layouts? assuming it doesn't get remove somehow 🤞

Comment (mcwinter07) [reply draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:87]: Unless this is to ensure specificity 🤔

File: draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss (hunk 4 above)
Comment (NatalieLoyzaga) [reply draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss:81]: bahaha my brain didn't put that together - wow

**Review by mcwinter07 — APPROVED** [review]

**Pull request merged**